docs/conf.py

```diff
@@ -67,8 +67,13 @@
     "IPython.sphinxext.ipython_console_highlighting",
     "sphinx_changelog",
     "plasmapy_sphinx",
+    "sphinxcontrib.bibtex",
 ]
 
+bibtex_bibfiles = ["bibliography.bib"]
+bibtex_default_style = "alpha"
```

Comment:
I think I actually prefer the `"plain"` style here, which just numbers bibliography.

For future reference though, this can be customized, see [`sphinxcontrib-bibtex` customizing section](https://sphinxcontrib-bibtex.readthedocs.io/en/latest/usage.html#bibtex-custom-formatting).  This customization actually leverages the [`pybtex.styles` API](https://docs.pybtex.org/api/styles.html#id4).

Reply:
I'm content to go with `"plain"` for the time being.  I'm not particularly fond of any of the built-in styles.  I would like us to eventually customize bibliography stuff to be a bit more like it is in plasma journals, but that's not a high priority.